src/IO/Observer/WriteSimpleData.hpp

```diff
@@ -25,8 +25,15 @@ namespace ThreadedActions {
 /*!
```

Comment:
These changes now make `WriteSimpleData` and `WriteReductionDataRow` basically the same, with the latter offering more functionality. It also removes the functionality to write to a `dat` file to a local `VolumeData` file. Please just switch to `WriteReductionDataRow` in CCE and revert the changes to `WriteSimpleData`.

Reply:
To be clear, I don't want to remove `WriteSimpleData`, I wouldn't be surprised if we want that functionality, at least for debugging, but I do want you to use the existing action that does what you want :)